appveyor.yml

```diff
@@ -1,28 +1,27 @@
-version: "{build}"
 environment:
   matrix:
-  - PYTHON: "C:\\Python27"
-  - PYTHON: "C:\\Python27-x64"
-  - PYTHON: "C:\\Python34"
-  - PYTHON: "C:\\Python34-x64"
-  - PYTHON: "C:\\Python35"
-  - PYTHON: "C:\\Python35-x64"
+  - PYTHON: 'C:\Python27'
+  - PYTHON: 'C:\Python27-x64'
+  - PYTHON: 'C:\Python34'
+  - PYTHON: 'C:\Python34-x64'
+  - PYTHON: 'C:\Python35'
+  - PYTHON: 'C:\Python35-x64'
 matrix:
   fast_finish: true
 init:
-- "ECHO %PATH%"
-- "ECHO %PYTHON%"
-- ps: "ls C:/Python*"
+- ps: ls C:/Python*
```

Comment:
we don't _need_ it, but it's nice to know what appveyor supports out of the box (was certainly useful when we were considering adding python3.5)